tutorials/helloworldwebapp/cleanup_app_and_pipelines.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+bold() {
+  echo ". $(tput bold)" "$*" "$(tput sgr0)";
+}
+
+pushd ~/spinnaker-for-gcp/
+
+source scripts/install/properties
+
+scripts/manage/check_project_mismatch.sh
+
+read -p ". $(tput bold)You are about to delete all resources from the helloworldwebapp application and pipelines. This step is not reversible. Do you wish to continue (Y/n)? $(tput sgr0)" yn
+case $yn in
+  [Yy]* ) ;;
+  "" ) ;;
+  * ) exit;;
+esac
+
+bold "Deleting CSR repository..."
+
+gcloud source repos delete spinnaker-marketplace-helloworldwebapp
+rm -rf ~/$PROJECT_ID/spinnaker-marketplace-helloworldwebapp
+
+bold "Deleting helloworldwebapp-prod and helloworldwebapp-staging Kubernetes resources..."
+
+kubectl delete -f tutorials/helloworldwebapp/templates/csr/config/staging/service.yaml
+kubectl delete -f tutorials/helloworldwebapp/templates/csr/config/prod/service.yaml
+
+bold "Deleting Cloud Build trigger..."
+
+for trigger in $(gcloud alpha builds triggers list --filter triggerTemplate.repoName=spinnaker-marketplace-helloworldwebapp --format 'get(id)'); do
+  gcloud alpha builds triggers delete -q $trigger
+done
+
+bold "Deleting GCR images..."
+
+for digest in $(gcloud container images list-tags gcr.io/${PROJECT_ID}/spinnaker-marketplace-helloworldwebapp --format='get(digest)'); do
+  gcloud container images delete -q --force-delete-tags "gcr.io/${PROJECT_ID}/spinnaker-marketplace-helloworldwebapp@${digest}"
+done
+
+bold "Deleting Spinnaker helloworldwebapp application and pipelines..."
+
+set -x
+~/spin pipeline delete -a helloworldwebapp -n "Deploy to Staging"
+~/spin pipeline delete -a helloworldwebapp -n "Deploy to Production"
+~/spin application delete helloworldwebapp
+{ set +x ;} 2> /dev/null
+
+popd
```

Comment:
If one copies/pastes their way through the entire tutorial, this final command results in:

```
Application deleted
/home/duftler/spinnaker-for-gcp/tutorials/helloworldwebapp/cleanup_app_and_pipelines.sh: line 50: popd: /home/duftler/rosco-oss-132/spinnaker-marketplace-helloworldwebapp: No such file or directory
.  Finished cleaning up helloworldwebapp resources. 
```

Reply:
Changed to just cd into `~/spinnaker-for-gcp`

Reply:
Actually, just changed the tutorial command so that users don't get stuck in a deleted repo